tajo-core/pom.xml

```diff
@@ -690,6 +696,37 @@
         </dependency>
       </dependencies>
     </profile>
+    <profile>
+      <id>parallel-test</id>
+      <activation>
+        <activeByDefault>false</activeByDefault>
+      </activation>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-surefire-plugin</artifactId>
```

Comment:
@ykrips 
If reuseForks is false, integration test is too many restarted the testing cluster
I've successfully ran the parallel execution with enabled reuseForks
```
mvn clean install -Pparallel-test,hcatalog-0.12.0 -DLOG_LEVEL=WARN -Dmaven.fork.count=3
```